Tests/scripts/validate_files.py

```diff
@@ -217,7 +217,22 @@ def validate_modified_files(self, modified_files, is_backward_check=True, old_br
                     self._is_valid = False
                 if not integration_validator.is_valid_integration():
                     self._is_valid = False
+            elif False and re.match(BETA_INTEGRATION_REGEX, file_path, re.IGNORECASE) or \
+                    re.match(BETA_INTEGRATION_YML_REGEX, file_path, re.IGNORECASE):
+                print("validating beta")
```

Comment:
do we really need this or is that leftover debug print?
```suggestion
                print("Validating beta")
```

Reply:
removed